.tmux.conf

```diff
@@ -1,49 +1,44 @@
+# bind -T copy-mode-vi v send-keys -X begin-selection
+# bind -T copy-mode-vi V send-keys -X rectangle-toggle
+# 
+# # Mouse options for tmux >= 2.5
+# set-option -g -q mouse on
+# bind-key -T root WheelUpPane if-shell -F -t = "#{alternate_on}" "send-keys -M" "select-pane -t =; copy-mode -e; send-keys -M"
+# bind-key -T root WheelDownPane if-shell -F -t = "#{alternate_on}" "send-keys -M" "select-pane -t =; send-keys -M"
+# bind-key -T copy-mode-vi WheelUpPane send-keys -X halfpage-up
```

Comment:
With current config `copy-mode-vi` wheel motion is the same with the exception of being 5 lines instead of `half page`